src/Features/Core/Portable/MoveToNamespace/AbstractMoveToNamespaceService.cs

```diff
@@ -252,24 +253,44 @@ private static async Task<ImmutableArray<ISymbol>> GetMemberSymbolsAsync(Documen
                 moveSpan,
                 MoveTypeOperationKind.MoveTypeNamespaceScope,
                 cancellationToken).ConfigureAwait(false);
-
             var modifiedDocument = modifiedSolution.GetDocument(document.Id);
-            var syntaxRoot = await modifiedDocument.GetSyntaxRootAsync(cancellationToken).ConfigureAwait(false);
 
+            // Since MoveTypeService doesn't handle linked files, we need to merge the diff ourselves, 
```

Comment:
Should MoveTypeService handle linked files?

Reply:
yes. And in my opinion, all services should handle linked files (similar to how ChangeNameSpace service works) as their results will be used as the base for further code change. Preferably, we should come up a shared service to handle this for them, which requires a bigger change. I will file an issue to track that.

Reply:
#40040